src/main/java/blusunrize/immersiveengineering/client/models/obj/OBJHelper.java

```diff
@@ -139,6 +139,7 @@
 		}
 	}
 
+	// TODO replace with Mixin duck?
```

Comment:
A lot of OBJHelper (and reflection into MC/Forge in general) can be done with Mixins instead, which would give compile-time guarantees that things will actually work. "Duck" in this case is a common term for an interface added to a class via a Mixin (similar to how our existing accessors work). Then you can pass around `MeshAccess`s (which would be the `ModelMesh`s themselves) or similar, instead of `Object`s in a thin reflection wrapper.